### ISSUE TITLE
SCM - Diff action buttons are too far from the gutter

### ISSUE THREAD

**Issue by yay** [microsoft/vscode#108278]
If I want to stage or undo a change, I click the gutter and drag my mouse cursor all the way to another side of the window each time, which is not ergonomic:

![vscode](https://user-images.githubusercontent.com/1799756/95399846-0b0dc080-0901-11eb-85a3-7e9e5860d7aa.gif)

There is a way to greatly streamline this experience by brining the git lens action buttons closer to the gutter:

![intellij](https://user-images.githubusercontent.com/1799756/95399852-106b0b00-0901-11eb-9818-117b2a34f127.gif)

Also notice how the diff for the added line is not even shown, because it's pointless. It would show the very same thing I'm looking at anyway.

In IntelliJ the diff only shows something I don't already see: changed or deleted lines. For example, this line has been changed, so the diff is shown:

<img width="321" alt="Screen Shot 2020-10-08 at 00 50 59" src="https://user-images.githubusercontent.com/1799756/95400422-7906b780-0902-11eb-94d8-ce7680be981e.png">

Can you please consider improving this?

**Comment by jrieken** [comment]
🏃

**Comment by joaomoreno** [comment]
@jrieken I take it you do not oppose this. What do you think about a peek widget option (actual boolean, not a user setting) to simply put actions on the left? I would always set this for the scm dirty diff peek instance.

**Comment by jrieken** [comment]
Yeah, maybe a simple left/right option. Tho, it might be super weird with non-diff actions, like close etc.
